lib/components/sprite_component.dart

```diff
@@ -23,14 +24,13 @@ class SpriteComponent extends PositionComponent {
   SpriteComponent();
 
   SpriteComponent.square(double size, String imagePath)
```

Comment:
since we have Vector2.all we could even drop this constructor. not sure, just an idea. the other one then could fromImagePath instead of reactangle